resources/ec2-vpc-peering-connections.go

```diff
@@ -0,0 +1,61 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+)
+
+type EC2VPCPeeringConnection struct {
+	svc *ec2.EC2
+	id  *string
+}
+
+func init() {
+	register("EC2VPCPeeringConnection", ListEC2VPCPeeringConnections)
+}
+
+func ListEC2VPCPeeringConnections(sess *session.Session) ([]Resource, error) {
+	svc := ec2.New(sess)
+	resources := make([]Resource, 0)
+
+	// filter should be set as deleted vpc connetions are returned
+	params := &ec2.DescribeVpcPeeringConnectionsInput{
+		Filters: []*ec2.Filter{
```

Comment:
For consistency, the filters should be applied in the `Filter` function, like done [with EC2 instances](https://github.com/rebuy-de/aws-nuke/blob/2215621/resources/ec2-instances.go#L42-L47).

As far as I remember we did it this way, so we can still see all resources in the output, regardless of it being filtered or not.

Reply:
Thanks for the feedback, I have amended accordingly.